server/services/datacommons.py

```diff
@@ -106,8 +90,9 @@ def point(entities, variables, date='', all_facets=False):
           observation is returned.
       all_facets (optional): Whether or not to get data for all facets.
   """
+  url = get_service_url('point')
```

Comment:
I'd suggest to use the full path like "/v1/bulk/observations/point" here and elsewhere instead of "point". This can make it very clear what is used and avoid using the wrong short  name.

Reply:
done.